# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/angle/checkout/src/compiler/preprocessor/DiagnosticsBase.cpp
# Commit: 3a91885d47e0
# Full Hash: 3a91885d47e0ef7fba9749562d52300eea67c651
# Author: Miko Mynttinen <mikokm@gmail.com>
# Date: 2019-11-11 21:52:52
# Regressor Bug: 1578576
# File Overlap Count: 4
# Description:
#   Bug 1578576 - Part 2: Update to ANGLE 3865 r=jgilbert
#   
#   Depends on D44561
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D44579
# ==============================================================================

diff -r 77cee77c0307 -r 3a91885d47e0 gfx/angle/checkout/src/compiler/preprocessor/DiagnosticsBase.cpp
--- a/gfx/angle/checkout/src/compiler/preprocessor/DiagnosticsBase.cpp	Fri Nov 08 16:41:42 2019 +0000
+++ b/gfx/angle/checkout/src/compiler/preprocessor/DiagnosticsBase.cpp	Fri Nov 08 17:06:00 2019 +0000
@@ -115,8 +115,8 @@
             return "invalid file number";
         case PP_INVALID_LINE_DIRECTIVE:
             return "invalid line directive";
-        case PP_NON_PP_TOKEN_BEFORE_EXTENSION_ESSL3:
-            return "extension directive must occur before any non-preprocessor tokens in ESSL3";
+        case PP_NON_PP_TOKEN_BEFORE_EXTENSION_ESSL:
+            return "extension directive must occur before any non-preprocessor tokens in ESSL";
         case PP_UNDEFINED_SHIFT:
             return "shift exponent is negative or undefined";
         case PP_TOKENIZER_ERROR:
@@ -129,7 +129,7 @@
             return "unexpected token after conditional expression";
         case PP_UNRECOGNIZED_PRAGMA:
             return "unrecognized pragma";
-        case PP_NON_PP_TOKEN_BEFORE_EXTENSION_ESSL1:
+        case PP_NON_PP_TOKEN_BEFORE_EXTENSION_WEBGL:
             return "extension directive should occur before any non-preprocessor tokens";
         case PP_WARNING_MACRO_NAME_RESERVED:
             return "macro name with a double underscore is reserved - unintented behavior is "